src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployService.kt

```diff
@@ -79,11 +78,25 @@ class DeployService(
         val deployId = UUID.randomUUID().toString()
 
         val openShiftResponses: List<OpenShiftResponse> = performOpenShiftCommands(deployId, deploymentSpec)
-        val performDeploy = !(deploymentSpec.deploy?.flags?.pause ?: false) && shouldDeploy
-        val redeployResponse: ResponseEntity<JsonNode>? = if (performDeploy) triggerRedeploy(deploymentSpec, openShiftResponses) else null
 
         val success = openShiftResponses.all { it.success }
-        return AuroraDeployResult(deployId, deploymentSpec, openShiftResponses, redeployResponse, success)
+        val noPause = !(deploymentSpec.deploy?.flags?.pause ?: false)
```

Comment:
In my opinion there is too much going on in relation to tagging and redeploying in this method. Should be extracted into own method.

In fact, the complexity of tagging and redeploying has grown to warrant its own class.